### DISCUSSION TITLE
Stacked in 2024 🦭

### DISCUSSION THREAD

**Discussion by FilledStacks** [FilledStacks/stacked#1053] [Announcements]
# What’s happening with Stacked?

I’ve received a few questions on Stacked, what’s happening with it, and it’s future so I want to answer it here. 

Stacked is at an amazing place, through years of testing in production. We’re at a point where we know the API, the community understands it, it’s stable and very reliable for teams that use it. 

I’m a firm believer in developing on a stable foundation, and because of that Stacked doesn’t see “rapid innovation” as a product. It is the foundation for professional Flutter developers to build on. 

The only Flutter framework that caters for every aspect of production development. I favour stability in developer tooling over shiny new features. 

Does that mean there will be no more development?

# How is Stacked being Developed?

**tl;dr -** Currently only critical bugs will be addressed with the occasional feature from our roadmap (see below)

I’ve changed the business model of FilledStacks, so I don’t have revenue to invest into full time development of Stacked. 

We are hard at work getting our products in the hands of teams that need it. [TestSweets](https://testsweets.carrd.co/), our no-code Flutter automation tool is built with Stacked, so we’ll be monitoring and improving Stacked through that development experience. 

[SessionMate](https://sessionmate.dev/), our enhanced crash reporting tool is built with Stacked. Through these products, we’re always using Stacked and will always be improving it. 

# What is the Future of Stacked?

There are a few things that I want to build into Stacked that I think will massively help us build functionality faster. 

### Website

- Improved landing page to make it feel more professional
- Testimonials from Stacked users - Please tweet and tag [Stacked](https://twitter.com/stacked_apps) with something nice, I want it on our homepage

### Docs

- More docs 😅

### CLI

- I want to update the template to include integration testing setup

### Features

- Lifecycle overrides: `onBackgroundEntered` , `onForegroundEntered` in the ViewModel
- First party support to trigger functions in ViewModels when something in a service changes

When will this work be done? Over time, there’s no clear date. These are all nice to have features and doesn’t change what you can build with Stacked, it will simply improve very small parts of Stacked. 

Thanks for reading, Enjoy using Stacked, and I’ll see you in the next update.
